noted/notes/v1/notes.proto

```diff
@@ -137,3 +144,12 @@ message DeleteBlockRequest {
 message DeleteBlockResponse {
 
 }
+
+message ExportNoteRequest {
+    string id = 1;
+    NoteExportFormat note_format = 2;
+}
+
+message ExportNoteResponse {
+    bytes note_bytes = 1;
```

Comment:
Naming it `note_bytes` is redundant I guess the type is already `bytes`. What's important is that these bytes constitue a file I guess. I suggest something like.
```suggestion
    bytes file = 1;
```